api/types/src/convert.rs

```diff
@@ -79,6 +80,24 @@ impl<'a, R: MoveResolverExt + ?Sized> MoveConverter<'a, R> {
         self.inner.view_resource(typ, bytes)?.try_into()
     }
 
+    pub fn try_into_resources_from_resource_group(
+        &self,
+        bytes: &[u8],
+    ) -> Result<Vec<MoveResource>> {
+        // Resource groups cannot be nested.
+        let resources_with_tag: Vec<(StructTag, Vec<u8>)> =
+            bcs::from_bytes::<BTreeMap<StructTag, Vec<u8>>>(bytes).map(|map| {
```

Comment:
if anything, it might have been useful to have a single file somewhere that had a `type ResourceGroup BTreeMap<StructTag, Vec<u8>>` or whatever the Rust syntax is.

Reply:
Updated